src/editorial/web/components/starRating/README.md

```diff
@@ -0,0 +1,3 @@
+# StarRating
+
+📣 For more context and visual guides relating to `StarRating` usage, visit the [Source Design System website](https://www.theguardian.design)
```

Comment:
Would you be able to include a code example here and a bit of API blurb?

Reply:
👍 3f95ec2